core/modules/system/system.install

```diff
@@ -3343,6 +3343,260 @@ function system_update_1085() {
   }
 }
 
+/**
+ * Activate the views admin listing of enabled modules.
+ */
+function system_update_1086() {
+  $view_config = array(
```

Comment:
Rather than duplicating all of this, I suggest you use `  config_install_default_config('system', 'views.view.list_modules');`

This function will essentially do the same thing you do here (check of the view exists, and if so, avoid installing it)

Reply:
There's a function for that!! :tada: 

Hm, nobody seems to have used that in install files so far - is that because it's not safe to use it in install, or because all the others also weren't aware of that function? :thinking: